portal/config.py

```diff
@@ -105,6 +105,9 @@ class BaseConfig(object):
 
     SYSTEM_TYPE = 'development'
 
+    SMARTLING_LANGUAGES = ['en_AU', 'es_MX', 'fr_CA']
+    SMARTLING_PROJECT_ID = '13f8e2dcf'
```

Comment:
I'm not sure we want to have separate projects for TrueNTH vs ePROMS. The files are uploaded to single projects in Smartling, so if we had multiple projects, it means we would be uploading message.po files for the same language to different projects, and doubling the amount of translation work needed (even if it's just copying/pasting all the translations across projects)

As for the languages, this is just the 'master list' of languages we have PO files for. The actual languages available to users are already restricted per site, based on the site orgs and their locales. So I would think the SMARTLING_LANGUAGES shouldn't be site-specific (especially if, as mentioned above, we only have one project for both sites anyway)